Look for libclang at the installed path for clang rather than the prefix for rust-bindgen
diff --git a/build/rust/rust_bindgen.gni b/build/rust/rust_bindgen.gni
index 5c809c6..782c198 100644
--- a/build/rust/rust_bindgen.gni
+++ b/build/rust/rust_bindgen.gni
@@ -19,7 +19,7 @@ if (host_os == "win") {
 
 # On Windows, the libclang.dll is beside the bindgen.exe, otherwise it is in
 # ../lib.
-_libclang_path = rust_bindgen_root
+_libclang_path = clang_base_path
 if (host_os == "win") {
   _libclang_path += "/bin"
 } else {
diff --git a/build/rust/rust_bindgen_generator.gni b/build/rust/rust_bindgen_generator.gni
index c91916b..e504824 100644
--- a/build/rust/rust_bindgen_generator.gni
+++ b/build/rust/rust_bindgen_generator.gni
@@ -18,7 +18,7 @@ if (host_os == "win") {
 
 # On Windows, the libclang.dll is beside the bindgen.exe, otherwise it is in
 # ../lib.
-_libclang_path = rust_bindgen_root
+_libclang_path = clang_base_path
 if (host_os == "win") {
   _libclang_path += "/bin"
 } else {
